apiserver/facades/agent/uniter/export_test.go

```diff
@@ -31,3 +32,7 @@ func NewStorageAPI(
 ) (*StorageAPI, error) {
 	return newStorageAPI(backend, storage, resources, accessUnit)
 }
+
+func SetNewContainerBrokerFucn(api *UniterAPI, newBroker caas.NewContainerBrokerFunc) {
```

Comment:
Should be `SetNewContainerBrokerFuck`.
I mean, `SetNewContainerBrokerFunc`.

Reply:
lol, fixed